### PR TITLE
Relnotes v2.0

### PR DESCRIPTION
**Pull request by jshaughn**
https://deploy-preview-829--kiali.netlify.app/news/release-notes/

https://deploy-preview-829--kiali.netlify.app/docs/installation/installation-guide/prerequisites/#version-compatibility

### CODE REVIEW

**Comment by jmazzitelli** [comment]
Either include in this PR (or have a separate PR) an FAQ that mentions how you can switch back to the old style graph (cytoscape).

Something in here https://kiali.io/docs/faq/graph/ like `How can I switch back to the old style (cytoscape) of the graph?`

File: content/en/news/release-notes.md [hunk 1]
```diff
@@ -7,46 +7,63 @@ weight: 1
...
+The first major Kiali release in over 5 years!  There are two main reasons for the major version update:
 
-* [Kiali federation with multiple Kubernetes flavors not able to access none OpenShift workloads/applications when running on OCP](https://github.com/kiali/kiali/issues/7665)
-* [Mesh page - controlplanes not mapped to their dataplanes when using stable revision labels](https://github.com/kiali/kiali/issues/7598)
-* [Validations - Inconsistency between List and Details pages](https://github.com/kiali/kiali/issues/7685)
-* [K8s Gateways: Out of mesh error when not in 'istio-system'](https://github.com/kiali/kiali/issues/7720)
-* [bearer token auth with external Grafana](https://github.com/kiali/kiali/issues/7717)
-* [grafana and tracing versions should be obtained over in_cluster_url](https://github.com/kiali/kiali/issues/7758)
-* [Toggle menu of the workload minigraph does not load the action list](https://github.com/kiali/openshift-servicemesh-plugin/issues/375)
+1. There is a breaking change in Kiali's namespace management configuration. To limit the namespaces accessible to Kiali, or made visible to users, iali v2.0 users will configure Discovery Selectors.
```
Comment (jmazzitelli) [inline content/en/news/release-notes.md]:
```suggestion
1. There is a breaking change in Kiali's namespace management configuration. To limit the namespaces accessible to Kiali, or made visible to users, Kiali v2.0 users will configure Discovery Selectors.
```

File: content/en/news/release-notes.md [hunk 2]
```diff
@@ -7,46 +7,63 @@ weight: 1
...
+  * api.namespaces.exclude
+  * api.namespaces.include
+  * api.namespaces.label_selector_exclude
+  * api.namespaces.label_selector_include
+
+2. Kiali has a new traffic graph implementation.
+
+The Cytoscape implementation has been deprecated and is no longer the default. Kiali has moved to PatternFly Topology to align with the rest of the Kiali interface, which is already implemented using PatternFly components.  The old graph implementation will be removed as soon as the Kiali maintainers believe the new implementation has proven itself in the field. Until that time, it can still be accessed by setting:
 
+  spec.kiali_feature_flags.ui_defaults.graph.impl: "cy"
```
Comment (jmazzitelli) [inline content/en/news/release-notes.md]: I would show this in a yaml block (or you can just wrap in single ticks to make it stand out like `spec.kiali_feature_flags.ui_defaults.graph.impl: "cy"`).
````suggestion
```yaml
spec:
  kiali_feature_flags:
    ui_defaults:
      graph:
        impl: "cy"
```
````

**Review by jmazzitelli — CHANGES_REQUESTED** [review]

**Comment by jshaughn** [comment]
@jmazzitelli , I added your FAQ.

**Review by jmazzitelli — APPROVED** [review]

**Pull request merged**